source/validation.php

```diff
@@ -154,5 +154,39 @@
     |
     */
 
-    'attributes' => [],
+    'attributes' => [
+        'address'               => 'Address',
+        'age'                   => 'Age',
+        'body'                  => 'Body',
+        'city'                  => 'City',
+        'content'               => 'Content',
+        'country'               => 'Country',
+        'date'                  => 'Date',
+        'day'                   => 'Day',
+        'description'           => 'Description',
+        'email'                 => 'Email',
+        'excerpt'               => 'Excerpt',
+        'first_name'            => 'First Name',
+        'gender'                => 'Gender',
+        'hour'                  => 'Hour',
+        'last_name'             => 'Last Name',
+        'message'               => 'Message',
+        'minute'                => 'Minute',
+        'mobile'                => 'Mobile',
+        'month'                 => 'Month',
+        'name'                  => 'Name',
+        'password'              => 'Password',
+        'password_confirmation' => 'Password Confirmation',
+        'phone'                 => 'Phone',
```

Comment:
Could you add the key `photo` that is used in [Laravel Jetstream](https://github.com/laravel/jetstream/blob/2.x/stubs/app/Actions/Fortify/UpdateUserProfileInformation.php#L24)?

Reply:
Agree. Added.

Reply:
See also https://github.com/Laravel-Lang/lang/pull/1862